TrackletAlgorithm/MemoryTemplateTPROJ.h

```diff
@@ -127,21 +142,21 @@ class MemoryTemplateTPROJ
   void clear()
   {
     DataType data("0",16);
-    MEM_RST: for (size_t ibx=0; ibx<(1<<NBIT_BX); ++ibx) {
+    MEM_RST: for (size_t ibx=0; ibx<DEPTH_BX; ++ibx) {
       mask_[ibx]=0;
       for (size_t page = 0; page < NPAGE; page++) {
-	nentries_[ibx*NPAGE+page] = 0;
-	for (size_t addr=0; addr<(1<<NBIT_ADDR); ++addr) {
-	  dataarray_[ibx][(1<<NBIT_ADDR)*page+addr] = data;
-	}
+      	nentries_[ibx*NPAGE+page] = 0;
+      	for (size_t addr=0; addr<DEPTH_ADDR; ++addr) {
+	        dataarray_[ibx][DEPTH_ADDR*page+addr] = data;
+      	}
```

Comment:
@tomalin Same comment regarding indentation.

```suggestion
        nentries_[ibx*NPAGE+page] = 0;
        for (size_t addr=0; addr<DEPTH_ADDR; ++addr) {
          dataarray_[ibx][DEPTH_ADDR*page+addr] = data;
        }
```